src/components/Match/MatchStory.jsx

```diff
@@ -562,6 +625,21 @@ const generateStory = (match) => {
     }
   }
 
+  // Remove any unneeded Time Markers
```

Comment:
This might be a more elegant solution:
http://stackoverflow.com/questions/30716829/how-to-remove-repeated-entries-from-an-array-while-preserving-non-consecutive-du

You can just check if the element before is a TimeMarketEvent